src/viadot/sources/sql_server.py

```diff
@@ -32,7 +32,7 @@ def __init__(
 
         super().__init__(*args, credentials=validated_creds, **kwargs)
         self.server = self.credentials.get("server")
-        self.username = self.credentials.get("usermame")
+        self.username = self.credentials.get("username")
```

Comment:
This can potentially break things; are you sure nobody depends on this typo? Might be good to communicate this to SQL Server source users (and also add to changelog).